CHANGELOG.md

```diff
@@ -14,6 +14,12 @@
 
   The prompt plugin now prints messages to `stderr` instead of `stdout`.
 
+### Bug fixes
```

Comment:
This might be a feature, though we probably should have just done this to begin with :woman_shrugging: